src/test/java/com/accelad/docx4j/TableInTableIntegrationTest.java

```diff
@@ -0,0 +1,90 @@
+package com.accelad.docx4j;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.apache.commons.io.IOUtils;
+import org.docx4j.openpackaging.exceptions.Docx4JException;
+import org.docx4j.openpackaging.packages.WordprocessingMLPackage;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.accelad.docx4j.facade.Paragraphs;
+import com.accelad.docx4j.facade.WordProcessor;
+import com.accelad.docx4j.facade.WordProcessorDocx4J;
+import com.accelad.docx4j.tags.ReportImage;
+import com.accelad.docx4j.tags.Tag;
+import com.accelad.docx4j.tags.TagName;
+import com.accelad.docx4j.tags.TagScanner;
+import com.accelad.docx4j.tags.TagUpdater;
+import com.accelad.docx4j.tags.TagUpdaterFactory;
+import com.accelad.docx4j.tags.TagValue;
+import com.accelad.docx4j.tags.TagValueImage;
+import com.accelad.docx4j.tags.TagValueString;
+import com.accelad.docx4j.tags.TagValueTable;
+import com.accelad.docx4j.tags.TagValues;
+
+public class TableInTableIntegrationTest {
+
+    private WordprocessingMLPackage wordprocessingMLPackage;
+    private Tag tag;
+
+    @Before
+    public void readFileAndGetTags() throws Docx4JException {
+        InputStream resourceAsStream = this.getClass()
+                .getResourceAsStream("/docx/test_with_tag_in_table.docx");
+        wordprocessingMLPackage = WordprocessingMLPackage.load(resourceAsStream);
+
+        WordProcessor processor = new WordProcessorDocx4J(wordprocessingMLPackage);
+        Paragraphs paragraphs = processor.getParagraphs();
+
+        tag = new TagScanner().scan(paragraphs).iterator().next();
+    }
+
+    @Test
+    public void should_update_tag_in_a_table_with_a_string() {
```

Comment:
Ok I see... Could we rename `wordprocessingMLPackage` to `WordProcessingTable` ?